ibm/data_source_ibm_appid_action_url.go

```diff
@@ -44,12 +45,13 @@ func dataSourceIBMAppIDActionURLRead(ctx context.Context, d *schema.ResourceData
 	tenantID := d.Get("tenant_id").(string)
 	action := d.Get("action").(string)
 
-	resp, _, err := appIDClient.GetCloudDirectoryActionURLWithContext(ctx, &appid.GetCloudDirectoryActionURLOptions{
+	resp, rawResp, err := appIDClient.GetCloudDirectoryActionURLWithContext(ctx, &appid.GetCloudDirectoryActionURLOptions{
 		TenantID: &tenantID,
 		Action:   &action,
 	})
 
 	if err != nil {
+		log.Printf("[DEBUG] Error getting AppID actionURL: %s\n%s", err, rawResp)
```

Comment:
log statements still requires TF_LOG to be enabled..
Instead please return so that it ll be easy to debug in schematics as well..
`return diag.Errorf("Error getting AppID actionURL: %s", err)` == > `return diag.Errorf("Error getting AppID actionURL:  %s\n%s", err, rawResp)`

Please make this change at all places..

Reply:
is it not possible to set TF_LOG variable in schematics? If we return full response we could potentially be printing out secret values, if user sets TF_LOG he more or less knows what he is doing

Reply:
I will make the change

Reply:
Done